### PR TITLE
Remove Django 1.7 support

### PR DESCRIPTION
**Pull request by maikhoepfel**
We only support two versions of Django. That policy might change given that
Django has started doing LTS releases. But for now, it only makes sense to
remove Django 1.7 support.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,19 +32,15 @@ env:
...
-        - DJANGO=Django==1.7.11
         - DJANGO=Django==1.8.8
         - DJANGO=Django==1.9.1
 
 
 matrix:
     exclude:
-        - python: 3.2
-          env: "DJANGO=Django==1.9.1"
+        # Python 3.3 is not supported by Django 1.9
```
Comment (john-parton) [inline .travis.yml]: Did you mean Python 3.2 isn't supported by Django 1.9?

**Comment by mvantellingen** [comment]
@maikhoepfel are you able to rebase this on master?

File: .travis.yml (hunk 1 above)
Comment (mvantellingen) [inline .travis.yml]: Django 1.9 only supports Python 3.4 and 3.5.

> Django 1.9 requires Python 2.7, 3.4, or 3.5. We highly recommend and only officially support the latest release of each series.
> 
> The Django 1.8 series is the last to support Python 3.2 and 3.3.

**Comment by mvantellingen** [comment]
Rebased on master

**Pull request merged**